db/migrate/20170324171408_initial_migration.rb

```diff
@@ -8,6 +8,7 @@ def change
 
     create_table :resources do |t|
       t.string :name, null: false
+      t.string :description
```

Comment:
It's mostly good practice. I sometimes edit migrations within a PR, but once they have been merged I open a new one.